whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisSearchServiceTestIntegration.java

```diff
@@ -216,6 +214,28 @@ public void search_accept_json() {
         assertThat(whoisObject.getPrimaryKey().get(0).getValue(), is("TP1-TEST"));
     }
 
+    @Test
+    public void search_accept_text_plain() {
+        databaseHelper.addObject(RpslObject.parse("" +
+                "aut-num:        AS102\n" +
+                "as-name:        End-User-2\n" +
+                "descr:          description\n" +
+                "admin-c:        TP1-TEST\n" +
+                "tech-c:         TP1-TEST\n" +
+                "mnt-by:         OWNER-MNT\n" +
+                "source:         TEST\n" +
+                "created:        2017-05-16T11:18:05Z\n" +
+                "last-modified:  2017-05-16T11:18:05Z"));
+        final String whoisResources = RestTest.target(getPort(), "whois/search?query-string=AS102&source=TEST")
+                .request(MediaType.TEXT_PLAIN)
+                .get(String.class);
+
+        //assertThat(whoisResources.getErrorMessages(), is(empty()));
```

Comment:
Add some asserts 😉  (they're commented out)

Reply:
Yes, I created the PRs without the tests, this part of code is just to help me to see the output